torch_glow/tests/nodes/sigmoid_test.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import torch
+import torch_glow
+
+from tests.utils import jitVsGlow
+
+# Basic test of the PyTorch sigmoid Node on Glow
+
+
+def test_sigmoid_basic():
+    def sigmoid_basic(a):
+        c = a.sigmoid()
+        return c.sigmoid()
+
+    x = torch.randn(6)
+
+    jitVsGlow(sigmoid_basic, x)
+
+def test_sigmoid_inplace():
+    def sigmoid_inplace(a):
+        c = a.sigmoid_()
+        return c.sigmoid_()
+
+    x = torch.randn(6)
+
+    jitVsGlow(sigmoid_inplace, x)
```

Comment:
We support inplace because we copy Glow's output back to `x`?

Reply:
In this case, I think the answer is yes, although I didnt see any using scenario